src/resultsdisassemblypage.cpp

```diff
@@ -199,3 +201,46 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
         m_objdump = QLatin1String("aarch64-linux-gnu-objdump");
     }
 }
+
+void ResultsDisassemblyPage::resetCallStack()
+{
+    m_callStack.clear();
+}
+
+void calcFunction(QString asmLineCall, QString* offset, QString* symName)
+{
+    QStringList sym = asmLineCall.trimmed().split(QLatin1Char('<'));
+    *offset = sym[0].trimmed();
+    *symName = sym[1].replace(QLatin1Char('>'), QLatin1String(""));
```

Comment:
dito, use value().trimmed() and then .remove() instead of replacing with an empty string